<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project description</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../melampig.qrc">
    <normaloff>:/monitor</normaloff>:/monitor</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Hostname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Timeout</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>SSL Method</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Alias</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="hostEdit"/>
          </item>
          <item>
           <widget class="QSpinBox" name="portSpin">
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>5432</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dbEdit">
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="timeoutSpin">
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sslCombo">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Disable</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Allow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Prefer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Require</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="aliasEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Test connection</string>
       </property>
       <property name="icon">
   <iconset resource="../../../melampig.qrc">
         <normaloff>:/icons/connect.png</normaloff>:/icons/connect.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
   <iconset resource="../../../melampig.qrc">
         <normaloff>:/icons/accept.png</normaloff>:/icons/accept.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
   <iconset resource="../../../melampig.qrc">
         <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hostEdit</tabstop>
  <tabstop>portSpin</tabstop>
  <tabstop>dbEdit</tabstop>
  <tabstop>timeoutSpin</tabstop>
  <tabstop>sslCombo</tabstop>
  <tabstop>testButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="./../../melampig.qrc"/>
 </resources>
 <connections/>
</ui>
